### ISSUE TITLE
serial print of integers

### ISSUE THREAD

**Issue by bmcage** [carlosperate/ardublockly#162]
The serial print only works on text, and as such on variables.

```
Blockly.Blocks['serial_print'] = {
 ...
        .setCheck(Blockly.Types.TEXT.checkList);

```  
Because of this, you cannot just print out what is read on an analog input. Or a number. You need to assign these to a variable, then print the variable. 

As you can print integers, why not extend what can be printed to avoid this workaround?

**Comment by carlosperate** [comment]
This is a very good point, thanks @bmcage I'll add it in the next update.

**Comment by carlosperate** [comment]
Fixed in  6edff27 by removing the block input type, so any block can be attached to serial.print().
